keyboards/sawnsprojects/re65/keymaps/vial/vial.json

```diff
@@ -0,0 +1,212 @@
+{
+  "name": "RE65",
+  "vendorId": "0x534C",
+  "productId": "0x0C65",
+  "matrix": {
+    "rows": 5,
+    "cols": 15
+  },
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_rgblight"],
+  "layouts": {
+    "labels": [
+      "Split Backspace",
+      ["Spacebar", "6.25U", "7U"],
+      "Rotary Encoder"
+    ],
+    "keymap": [
+  [
+    {
+      "y": 0.25,
+      "x": 13,
+      "c": "#aaaaaa"
+    },
+    "0,13\n\n\n0,1",
+    "2,13\n\n\n0,1",
+    {
+      "c": "#777777"
+    },
+    "0,14\n\n\n2,1\n\n\n\n\n\ne0"
+  ],
+  [
+    {
+      "y": 0.25
+    },
+    "0,0",
+    {
+      "c": "#cccccc"
+    },
+    "0,1",
+    "0,2",
+    "0,3",
+    "0,4",
+    "0,5",
+    "0,6",
+    "0,7",
+    "0,8",
+    "0,9",
+    "0,10",
+    "0,11",
+    "0,12",
+    {
+      "c": "#aaaaaa",
+      "w": 2
+    },
+    "0,13\n\n\n0,0",
+    {
+      "c": "#cccccc"
+    },
+    "0,14\n\n\n2,0"
+  ],
+  [
+    {
+      "c": "#aaaaaa",
+      "w": 1.5
+    },
+    "1,0",
+    {
+      "c": "#cccccc"
+    },
+    "1,1",
+    "1,2",
+    "1,3",
+    "1,4",
+    "1,5",
+    "1,6",
+    "1,7",
+    "1,8",
+    "1,9",
+    "1,10",
+    "1,11",
+    "1,12",
+    {
+      "w": 1.5
+    },
+    "1,13",
+    "1,14"
+  ],
+  [
+    {
+      "c": "#aaaaaa",
+      "w": 1.75
+    },
+    "2,0",
+    {
+      "c": "#cccccc"
+    },
+    "2,1",
+    "2,2",
+    "2,3",
+    "2,4",
+    "2,5",
+    "2,6",
+    "2,7",
+    "2,8",
+    "2,9",
+    "2,10",
+    "2,11",
+    {
+      "c": "#777777",
+      "w": 2.25
+    },
+    "2,12",
+    {
+      "c": "#cccccc"
+    },
+    "2,14"
+  ],
+  [
+    {
+      "c": "#aaaaaa",
+      "w": 2.25
+    },
+    "3,0",
+    {
+      "c": "#cccccc"
+    },
+    "3,2",
+    "3,3",
+    "3,4",
+    "3,5",
+    "3,6",
+    "3,7",
+    "3,8",
+    "3,9",
+    "3,10",
+    "3,11",
+    {
+      "c": "#aaaaaa",
+      "w": 1.75
+    },
+    "3,12",
+    {
+      "c": "#777777"
+    },
+    "3,13",
+    {
+      "c": "#cccccc"
+    },
+    "3,14"
+  ],
+  [
+    {
+      "c": "#aaaaaa",
+      "w": 1.25
+    },
+    "4,0\n\n\n1,0",
+    {
+      "w": 1.25
+    },
+    "4,1\n\n\n1,0",
+    {
+      "w": 1.25
+    },
+    "4,2\n\n\n1,0",
+    {
+      "c": "#cccccc",
+      "w": 6.25
+    },
+    "4,6\n\n\n1,0",
+    {
+      "c": "#aaaaaa",
+      "w": 1.25
+    },
+    "4,10\n\n\n1,0",
+    {
+      "w": 1.25
+    },
+    "4,11\n\n\n1,0",
+    {
+      "x": 0.5,
+      "c": "#777777"
+    },
+    "4,13",
+    "4,14",
+    "4,14"
+  ],
+  [
+    {
+      "y": 0.25,
+      "c": "#aaaaaa",
+      "w": 1.5
+    },
+    "4,0\n\n\n1,1",
+    "4,1\n\n\n1,1",
+    {
+      "w": 1.5
+    },
+    "4,2\n\n\n1,1",
+    {
+      "c": "#cccccc",
+      "w": 7
+    },
+    "4,6\n\n\n1,1",
+    {
+      "c": "#aaaaaa",
+      "w": 1.5
+    },
+    "4,11\n\n\n1,1"
+  ]
+]
```

Comment:
```suggestion
    "keymap": [
      [
        {
          "x": 14.5,
          "c": "#777777"
        },
        "0,0\n\n\n2,1\n\n\n\n\n\ne",
        "0,1\n\n\n2,1\n\n\n\n\n\ne"
      ],
      [
        {
          "x": 13,
          "c": "#aaaaaa"
        },
        "0,13\n\n\n0,1",
        "2,13\n\n\n0,1"
      ],
      [
        {
          "y": 0.25,
          "c": "#777777"
        },
        "0,0",
        {
          "c": "#cccccc"
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        "0,9",
        "0,10",
        "0,11",
        "0,12",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "0,13\n\n\n0,0",
        {
          "c": "#cccccc"
        },
        "0,14\n\n\n2,0"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "1,0",
        {
          "c": "#cccccc"
        },
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "w": 1.5
        },
        "1,13",
        "1,14"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "2,10",
        "2,11",
        {
          "c": "#777777",
          "w": 2.25
        },
        "2,12",
        {
          "c": "#cccccc"
        },
        "2,14"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "3,0",
        {
          "c": "#cccccc"
        },
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        "3,11",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "3,12",
        {
          "c": "#777777"
        },
        "3,13",
        {
          "c": "#cccccc"
        },
        "3,14"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "4,0\n\n\n1,0",
        {
          "w": 1.25
        },
        "4,1\n\n\n1,0",
        {
          "w": 1.25
        },
        "4,2\n\n\n1,0",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "4,6\n\n\n1,0",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "4,10\n\n\n1,0",
        {
          "w": 1.25
        },
        "4,11\n\n\n1,0",
        {
          "x": 0.5,
          "c": "#777777"
        },
        "4,13",
        "4,14",
        "4,14"
      ],
      [
        {
          "y": 0.25,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "4,0\n\n\n1,1",
        "4,1\n\n\n1,1",
        {
          "w": 1.5
        },
        "4,2\n\n\n1,1",
        {
          "c": "#cccccc",
          "w": 7
        },
        "4,6\n\n\n1,1",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "4,11\n\n\n1,1"
      ]
    ]
```

Current JSON in this state, the `Rotary Encoder` option doesn't work because it's done in the `v3` style for VIA V3 firmware.
![image](https://github.com/vial-kb/vial-qmk/assets/100170946/f6fcdfb1-afa0-4e38-9b1b-7951aefcfd77)